packages/emails/src/templates/index.ts

```diff
@@ -18,3 +18,4 @@ export { OrganizerRescheduledEmail } from "./OrganizerRescheduledEmail";
 export { OrganizerScheduledEmail } from "./OrganizerScheduledEmail";
 export { TeamInviteEmail } from "./TeamInviteEmail";
 export { BrokenIntegrationEmail } from "./BrokenIntegrationEmail";
+export * from "@calcom/app-store/ee/routing-forms/emails/components";
```

Comment:
Import all email components here so that `renderEmail` method supports routing-forms email components as welll